lib/rubocop/cop/lint/or_assignment_to_constant.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Lint
+      # This cop checks for unintended or-assignment to a constant.
+      #
+      # Constants should always be assigned in the same location. And its value
+      # should always be the same. If constants are assigned in multiple
+      # locations, the result may vary depending on the order of `require`.
+      #
+      # Also, if you already have such an implementation, auto-correction may
+      # change the result.
+      #
+      # @example
+      #
+      #   # bad
+      #   CONST ||= 1
+      #
+      #   # good
+      #   CONST = 1
+      #
+      class OrAssignmentToConstant < Base
+        extend AutoCorrector
+
+        MSG = 'Avoid use or-assignment with constants.'
```

Comment:
```suggestion
        MSG = 'Avoid using or-assignment with constants.'
```

Sorry, I made a typo in my previous suggestions! 🤦

Reply:
I applied this commit-suggestion and fixed related spec :+1: